WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -628,6 +640,23 @@ private extension StorePickerViewController {
         fancyAlert.transitioningDelegate = AppDelegate.shared.tabBarController
         present(fancyAlert, animated: true)
     }
+
+    func displayExpiredWPComPlanAlert(siteID: Int64) {
+        let alertController = UIAlertController(title: Localization.ExpiredWPComPlanAlert.title,
+                                                message: Localization.ExpiredWPComPlanAlert.message,
+                                                preferredStyle: .alert)
+        let action = UIAlertAction(title: Localization.ExpiredWPComPlanAlert.upgrade, style: .default) { [weak self] _ in
+            if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.freeTrialInAppPurchasesUpgradeM1) {
+                let upgradesController = UINavigationController(rootViewController: UpgradesHostingController(siteID: siteID))
```

Comment:
I don't _think_ this should be in a `UINavigationController` – it's designed and expected to be shown as a modal containing the `HostingController` directly, and has a custom top bar and its own `NavigationView`.

Reply:
Good catch! Updated in 5dbd001d1cd3796a976931e711744f74eb4b0513.